nservicebus/extending/queuecreation.md

```diff
@@ -0,0 +1,35 @@
+---
```

Comment:
a new directory? most of our apis are about "extending" i dont see why this page gets its own directory